data-raw/standard_format_markdowns/cordoleani_data.Rmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Flora Cordoleani Data"
+author: "Maddee Rubenson (FlowWest)"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      warning = FALSE, 
+                      message = FALSE)
+
+library(tidyverse)
+library(lubridate)
+library(knitr)
+library(readxl)
+library(sf)
+library(preyDataProcessing)
+
+options(scipen=999)
+```
+
+## Cordoleani Data Standardization
+
+**Datasets provided:**
+
+-   `cage_locations.csv`
+-   `cagesalmon_growth.csv`
+-   `dailycagefish_data_2019.csv`
+-   `dailycagefish_data_2020.csv`
+-   `doboall.csv`
+-   `tidy_zoop_density.csv`
+
+**Author contact info:**
+
+Flora Cordoleani [[flora.cordoleani\@noaa.gov](flora.cordolean@noaa.gov)]
+
+### Prey Data
+
+Final prey density dataset includes the following variables:
+
+-   `date`: YYYY-MM-DD
+-   `gear_type`: the type of gear used to collect zooplankton.
+-   `species`: species of zooplankton
+-   `prey_density`: density of zooplankton (count/L)
+-   `size_class`: size class of zooplankton, determined by mesh size
+-   `mesh_size`: mesh size of net used to collect zooplankton
+-   `habitat_type`: habitat type of location where zooplankton were collected
+-   `lat`: latitude of sampling location
+-   `lon`: longitude of sampling location
+-   `site`: location description
+-   `author`: author of dataset
+-   `watershed`: unique watershed name associated with collected data
+
+#### Raw data
+
+```{r}
+cordo_prey_raw <- read_csv('../cordoleani/tidy_zoop_density.csv') |> glimpse()
```

Comment:
please use standard paths based on the package base directory 

"data-raw/cordoleani/tidy_zoop_density.csv"